src/lib.rs

```diff
@@ -317,67 +317,32 @@ impl<A: Send + 'static> Cancel<A> for Option<A> {
     }
 }
 
-/*
- *
- * ===== Async implementations =====
- *
- */
-
-macro_rules! async_impl_body {
-    ($ty:ty) => (
-        fn is_ready(&self) -> bool {
-            true
-        }
+/// Convenience implementation for (), to ease use of side-effecting functions returning unit
```

Comment:
It's unit. Void is usually reserved for the empty type `enum Void { }`, which has no values at all.